_includes/carbonads.html

```diff
@@ -1,13 +1,8 @@
-<div id="carbonads-container">
-  <div class="carbonad">
-    <div id="azcarbon"></div>
-    <script>
-      var z = document.createElement("script");
-      z.type = "text/javascript";
-      z.async = true;
-      z.src = "https://engine.carbonads.com/z/32340/azcarbon_2_1_0_VERT";
-      var s = document.getElementsByTagName("script")[0];
-      s.parentNode.insertBefore(z, s);
-    </script>
-  </div>
-</div>
+<script>
+  var z = document.createElement("script");
+  z.type = "text/javascript";
+  z.async = true;
+  z.src = "https://engine.carbonads.com/z/32340/azcarbon_2_1_0_VERT";
+  var s = document.getElementsByTagName("script")[0];
+  s.parentNode.insertBefore(z, s);
```

Comment:
Any reason why we don't use `document.head. appendChild` here?

Reply:
This is their snippet, not ours.